client/checkout/api/index.js

```diff
@@ -662,6 +662,7 @@ export default class WCPayAPI {
 				_wpnonce: nonce,
 				email: userEmail,
 				user_session: platformCheckoutUserSession,
+				return_url: window?.location?.href ?? '',
```

Comment:
The defensiveness here with the nullable operators isn't required but I thought it wouldn't hurt so I went ahead with it anyway, just in case.